src/gvpn_controller.py

```diff
@@ -110,7 +117,7 @@ def create_connection_req(self, data):
         self.ondemand_create_connection(uid, send_req=True)
 
     def serve(self):
-        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"])
+        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"]) # Select sockets ready to be served
         for sock in socks:
```

Comment:
@kyuhojeong @pstjuste Is it just me, or is this loop redundant? Since `serve()` is being called in a while-loop, if we instead only did `sock = socks[0]`, all the sockets would eventually get handled anyways, because the next socket would be marked as immediately available upon the next call to `select(...)`.

If we could get rid of a level of indentation here, I think it would be a big win for readability, because this is a long method.